modules/navigation/navigation_mesh_generator.cpp

```diff
@@ -489,7 +489,7 @@ NavigationMeshGenerator::~NavigationMeshGenerator() {
 }
 
 void NavigationMeshGenerator::bake(Ref<NavigationMesh> p_nav_mesh, Node *p_node) {
-	ERR_FAIL_COND(!p_nav_mesh.is_valid());
+	ERR_FAIL_COND_MSG(!p_nav_mesh.is_valid(), "Invalid Navigation Mesh");
```

Comment:
Add a period at the end, use sentence case:

```suggestion
	ERR_FAIL_COND_MSG(!p_nav_mesh.is_valid(), "Invalid navigation mesh.");
```

Reply:
Updated